SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -72,6 +72,10 @@ public ProcessRunnerArguments(string exeName, ILogger logger)
 
         public ILogger Logger { get { return this.logger; } }
 
+        #endregion Public properties
+
+        #region Public Methods
+
         public string GetQuotedCommandLineArgs()
         {
             if (this.CmdLineArgs == null) { return null; }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change 'ProcessRunnerArguments.GetQuotedCommandLineArgs()' to a property if appropriate. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AUsePropertiesWhereAppropriate)